src/lib/fileutils.ts

```diff
@@ -161,6 +161,18 @@ export const generateServiceBuildAndUpdatePipelineYaml = (
   );
 };
 
+/**
+ * Sanitize the given path to format Azure DevOps can properly utilize
+ *
+ * Transforms:
+ * - If present, removes leading dot-slash (`./`) prefix from the path
+ *
+ * @param pathLike a path-like string to sanitize
+ */
+export const sanitizeTriggerPath = (pathLike: string): string => {
+  return pathLike.replace(new RegExp("^./"), "");
```

Comment:
i think it should be
```
return pathLike.replace(/^\.\//, "");
```
because . means anything in regular expression hence it needs to be escaped

Reply:
woops. Forgot what i needed to escape since I was using `new RegExp` instead of `//`. Changed and added another test.